packages/cli/src/utils/memory.js

```diff
@@ -0,0 +1,13 @@
+import chalk from 'chalk'
+import consola from 'consola'
+import prettyBytes from 'pretty-bytes'
+
+export function getMemoryUsage() {
+  // https://nodejs.org/api/process.html#process_process_memoryusage
+  const { heapUsed, rss } = process.memoryUsage()
+  return `Memory usage: ${chalk.bold(prettyBytes(heapUsed))} (RSS: ${prettyBytes(rss)})`
```

Comment:
About `getMemoryUsage`, I would return an object instead of a string directly:

```suggestion
  return { heap: heapUsed, rss }
```